src/extensions/timeseries/TimeseriesViewNew/TimeseriesWidgetNew.tsx

```diff
@@ -174,22 +172,19 @@ const TimeseriesWidgetNew = (props: Props) => {
     }, [recordingSelectionDispatch])
 
     useEffect(() => {
-        if ((timeseriesData !== prevTimeseriesData) || (visibleChannelIds !== prevVisibleChannelIds)) {
-            // we only want to do this once (as a function of the timeseries data)
-            const panels0: Panel[] = []
-            for (let ch = 0; ch < timeseriesData.numChannels(); ch ++) {
-                // todo: i guess we need to redefine the panels whenever y_offsets or y_scale_factor or channel_ids change
-                const channel_id = channel_ids[ch]
-                if ((!visibleChannelIds) || (visibleChannelIds.includes(channel_id))) {
-                    const p = new Panel(ch, channel_id, timeseriesData, 0, y_scale_factor) // y_offsets[ch] replaced with 0
-                    panels0.push(p)
-                }
+        const panels0: Panel[] = []
+        for (let ch = 0; ch < timeseriesData.numChannels(); ch ++) {
+            // todo: i guess we need to redefine the panels whenever y_offsets or y_scale_factor or channel_ids change
+            const channel_id = channel_ids[ch]
+            if ((!visibleChannelIds) || (visibleChannelIds.includes(channel_id))) {
```

Comment:
here `visibleChannelIds` would indicate the set of selected electrodes? So show everything if nothing's selected, otherwise show only what's selected?